oop_bank_account/razi.py

```diff
@@ -0,0 +1,28 @@
+
+class Account :
+    def __init__(self, name, balance):
+        self.AccOwner=name
```

Comment:
Usually we use lower case letters for class attributes, such as:
`self.acc_owner = name`